python/ray/experimental/data/datasource/parquet_datasource.py

```diff
@@ -40,6 +40,8 @@ def prepare_read(
             paths, filesystem)
         file_sizes = [file_info.size for file_info in file_infos]
 
+        filesystem = _maybe_wrap_fs(filesystem)
```

Comment:
This doesn't do anything; you see the read tasks below have a different workaround implemented from https://github.com/ray-project/ray/pull/17492

Please revert.